samples/terminal/src/main/java/com/kody/ExampleSendPayment.java

```diff
@@ -1,67 +1,53 @@
 package com.kody;
 
-import com.kodypay.grpc.pay.v1.*;
-import io.grpc.ManagedChannel;
+import com.kodypay.grpc.pay.v1.KodyPayTerminalServiceGrpc;
+import com.kodypay.grpc.pay.v1.PayRequest;
+import com.kodypay.grpc.pay.v1.PayResponse;
 import io.grpc.ManagedChannelBuilder;
 import io.grpc.Metadata;
 import io.grpc.stub.MetadataUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
 public class ExampleSendPayment {
-    private static final Logger LOG = LoggerFactory.getLogger(ExampleSendPayment.class);
-    private static final long TIMEOUT_MS = java.time.Duration.ofMinutes(3).toMillis();
+    //TODO: Replace this with the testing or live environment
+    public static final String HOSTNAME = "grpc-staging.kodypay.com";
+    public static final String API_KEY = "API KEY";
 
     public static void main(String[] args) {
-        // Load configuration properties
-        Properties properties = loadProperties();
-        var address = properties.getProperty("address", "grpc-staging.kodypay.com");
-        var apiKey = properties.getProperty("apiKey");
-        var terminalId = properties.getProperty("terminalId");
-        if (apiKey == null) {
-            throw new IllegalArgumentException("Environment variable API-KEY is missing");
-        }
-        var storeId = properties.getProperty("storeId");
-
-        // Initialize PaymentClient
-        Metadata metadata = new Metadata();
-        metadata.put(Metadata.Key.of("X-API-Key", Metadata.ASCII_STRING_MARSHALLER), apiKey);
-
-        ManagedChannel channel = ManagedChannelBuilder
-                .forAddress(address, 443)
-                .idleTimeout(TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .keepAliveTimeout(TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .intercept(MetadataUtils.newAttachHeadersInterceptor(metadata))
-                .build();
-
-        KodyPayTerminalServiceGrpc.KodyPayTerminalServiceBlockingStub paymentClient = KodyPayTerminalServiceGrpc.newBlockingStub(channel);
+        //TODO: Replace this with your Store ID
+        String storeId = "STORE ID";
+        //TODO: Replace this with your Terminal ID
+        String terminalId = "TERMINAL ID";
+        //TODO: Replace this with your amount
+        String amount = "2000";
```

Comment:
as above, use the numeric type here and convert to string in the function below.
also explain in the comments what the type should be if we use minor units